docs/Deprecating-Disabling-and-Removing-Casks.md

```diff
@@ -0,0 +1,110 @@
+
+Unpopular casks (i.e. have fewer than 1000 [analytics installs in the last 30 days](https://formulae.brew.sh/analytics/cask-install/30d/)) can be deprecated immediately for any of the reasons above e.g. they cannot be installed.
+
+**Note: disabled casks in `homebrew/cask` will be automatically removed one year after their disable date.**
+
+Unpopular casks (i.e. have fewer than 1000 [analytics installs in the last 30 days](https://formulae.brew.sh/analytics/cask-install/30d/)) can be manually removed three months after their disable date.
+
+To disable a cask, add a `disable!` call. This call should include a deprecation date (in the ISO 8601 format) and a deprecation reason:
+
+```ruby
+disable! date: "YYYY-MM-DD", because: :reason
+```
+
+The `date` parameter should be set to the date that the reason for disabling came into effect. If there is no clear date but the cask needs to be disabled, use today's date. If the `date` parameter is set to a date in the future, the cask will be deprecated until that date (on which the cask will become disabled).
+
+The `because` parameter can be a preset reason (using a symbol) or a custom reason. See the [Deprecate and Disable Reasons](#deprecate-and-disable-reasons) section below for more details about the `because` parameter.
+
+## Removal
+
+A cask should be removed if it does not meet our criteria for [acceptable casks](Acceptable-Casks.md), does not install, or has been disabled for over a year.
+
+## Deprecate and Disable Reasons
+
+When a cask is deprecated or disabled, a reason explaining the action must be provided.
+
+There are two ways to indicate the reason. The preferred way is to use a pre-existing symbol to indicate the reason. The available symbols are listed below and can be found in the [`DeprecateDisable` module](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/deprecate_disable.rb):
+
+- `:does_not_build`: the cask cannot be installed on any supported macOS version.
+- `:repo_archived`: the upstream repository has been archived and no replacement is pointed to that we can use
+- `:repo_removed`: the upstream repository has been removed and no replacement is mentioned on the homepage that we can use
+- `:unmaintained`: the project appears to be abandoned i.e. it has had no commits for at least a year and has critical bugs or CVE that have been reported and gone resolved longer. Note: some software is "done"; a lack of activity does not imply a need for removal.
+- `:unsupported`: Homebrew's compilation of the software is not supported by the upstream developers (e.g. upstream only supports macOS versions older than 10.15)
+- `:deprecated_upstream`: the project is deprecated upstream and no replacement is pointed to that we can use
+- `:checksum_mismatch`: the checksum of the source for the casks's current version has changed since bottles were built and we cannot find a reputable source or statement justifying this
```

Comment:
Based on https://github.com/Homebrew/brew/blob/master/Library/Homebrew/deprecate_disable.rb, it looks like the only option for casks is `:discontinued`. This should likely be expanded to include some additionals, for example `:download_unavailable` or `:url_unavailable`

Reply:
Agreed.  We need some additional reasons as mentioned.

Reply:
@bevanjkay Misread that. Agreed we should add more.

Reply:
Decision at the time was that the formula list was bloated and didn't always apply to casks.

Agreed that we can add additional reasons though this should be a separate PR.

Reply:
The docs do need to be trimmed down to only reference `:discontinued` for the Casks section.
I'm happy to look at expanding the other potential reasons next week when I'm back at home.